src/dataServices/dynamoDbDataService.ts

```diff
@@ -385,6 +385,26 @@ export class DynamoDbDataService implements Persistence, BulkDataAccess {
         return exportJob;
     }
 
+    async getActiveSubscriptions(params: { tenantId?: string }): Promise<Record<string, any>[]> {
+        this.assertValidTenancyMode(params.tenantId);
+        let keyCondition = '_subscriptionStatus = :active';
+        if (params.tenantId) {
+            keyCondition += ` AND id beginsWith ${params.tenantId}`;
+        }
+        const subscriptionQuery = {
+            TableName: RESOURCE_TABLE,
+            IndexName: 'activeSubscriptions',
+            KeyConditionExpression: keyCondition,
+        };
```

Comment:
Did you test against DDB?

I think that this will fail since `:active` is missing in `ExpressionAttributeValues`.

See an example at: 
https://github.com/awslabs/fhir-works-on-aws-persistence-ddb/blob/e2ccf527eeb5fede5ac0a1b42511ec4ab08b7255/src/dataServices/dynamoDbParamBuilder.test.ts#L286-L297

Reply:
I changed the pattern of this function to match the others, using parambuilder and wrote more tests on each part of the implementation.